pages/vi/profiles/jay-jhanji.md

```diff
@@ -0,0 +1,44 @@
+#<center> Jay Jhanji </center>
```

Comment:
please do not use HTML :)

Reply:
Sorry, just couldnt resist. I have made the changes. Kindly review again. Thank you